src/config/passport.js

```diff
@@ -0,0 +1,45 @@
+import passport from 'passport';
+import {
+  Strategy as GoogleStrategy
+} from 'passport-google-oauth20';
+import env from './env-config';
+import Helpers from '../utils/helpers';
+
+const FacebookStrategy = require('passport-facebook').Strategy;
+
+const {
+  passportCb
+} = Helpers;
+
+const {
+  GOOGLE_CLIENT_ID,
+  GOOGLE_CLIENT_SECRET,
+  FACEBOOK_CLIENT_ID,
+  FACEBOOK_CLIENT_SECRET
+} = env;
+
+const setupPassport = () => {
+  passport.use(
+    new GoogleStrategy({
+      callbackURL: 'http://localhost:3000/api/v1/auth/google/redirect',
```

Comment:
you will want to use an env variable for this. so that it would be easy to set the callback URL in production